lib/jekyll/watcher.rb

```diff
@@ -40,7 +40,32 @@ def watch(options, site = nil)
       # You pressed Ctrl-C, oh my!
     end
 
-    # TODO: shouldn't be public API
+    # Paths to ignore for the watch option
+    #
+    # options - A Hash of options passed to the command
+    #
+    # Returns a list of relative paths from source that should be ignored
+    def listen_ignore_paths(options)
```

Comment:
`listen_ignore_paths` seems to need to be public for CI to pass, but nothing else needs to be public.
If we make them private, it allows us to change method signatures without bumping major version.

Reply:
Is there something to change in the tests for private methods?

Reply:
That exceeds my current Ruby knowledge. It can be done in Java; I imagine there is a way to do it in Ruby. I will investigate when I get a chance.

Reply:
Got it :+1: